src/pages/tag/index.astro

```diff
@@ -0,0 +1,67 @@
+---
+import PageLayout from "@/layouts/PageLayout.astro"
+import TagDictCard from "@/components/TagDictCard.astro"
+import { getAllEpisodeTags } from '@/helpers'
+
+const pageTitle = 'All episode tags'
+const tagDict = await getAllEpisodeTags(true) as TagDictionary
+  const tagDictAsList = Object.entries(tagDict).map(([letter, tagCountsMap]) => ({ letter, tagCountsMap }))
```

Comment:
There's an extra unnecessary space in front of the `const` on line 8 here, can you remove?